README.md

```diff
@@ -26,26 +34,42 @@ If you are using Maven to manage your project's dependencies, you can run `mvn i
 </dependency>
 ```
 
-If you aren't using Maven, you can add the `jar` in the normal way. This `jar` can be found in
-the `target` directory created when you ran `mvn package`.
+_NOTE:_ Depending on what version of the Java compiler you have, you may need to update the version. At this time, the {version} most compatible is `1.8`:
+
+```xml
+<properties>
+    <maven.compiler.source>{version}</maven.compiler.source>
+    <maven.compiler.target>{version}</maven.compiler.target>
+</properties>
+```
+
+3. Run `mvn install` to install packages and their dependencies locally.
 
-Note that we ran `mvn package` without running the integration tests, since these require access to a Conjur instance. You can run the
-integration tests with `mvn package` once you finished with the configuration.
+_NOTE:_ that we ran mvn package without running the integration tests, since these require access to a Conjur instance. You can run the integration tests 
```

Comment:
This should get put with the actual command that includes `-DskipTests`, and just refer people to `#contributing` if they want more info about how to run the tests.